pkg/GAPJulia/JuliaInterface/julia/gap_to_julia.jl

```diff
@@ -30,10 +30,13 @@ function gap_to_julia(t::T, x::Any) where T <: Type
 end
 
 ## Default
-gap_to_julia(::Type{Any},         x::GAPInputType) = gap_to_julia(x)
-gap_to_julia(::Type{Any},         x::Any         ) = x
-gap_to_julia(::T,                 x::Nothing     ) where T <: Type = nothing
-gap_to_julia(::Type{Any},         x::Nothing     ) = nothing
+gap_to_julia(::Type{Any},         x::MPtr   ) = gap_to_julia(x)
+gap_to_julia(::Type{Any},         x::FFE    ) = x
+gap_to_julia(::Type{Any},         x::Int64  ) = x
+gap_to_julia(::Type{Any},         x::Bool   ) = x
```

Comment:
Note that here, we also avoid the overhead of dispatching to `gap_to_julia`, and directly can write `x`. (Though I'd hope the Julia compiler would have optimized that call away anyway; but this way, also anybody reading the code, e.g. me, understands what's going on with one glance).

Reply:
Agreed. The last thing I could add here is this
```suggestion
gap_to_julia(::Type{Any},         x::Union{Bool,FFE,Int64}   ) = x
```
instead of the last three lines

Reply:
Fair enough. I'll update the PR.

Reply:
Actually, it turns out that this rule simply was redundant, so I removed it. Please take a look. If you agree, then this PR is ready for merging.